CONTRIBUTING.md

```diff
@@ -98,7 +98,7 @@ presented.
    practice to do before coding more intricate changes.
 
 In addition to the above, there are many pages on [our
-Wiki](https://github.com/flutter/flutter/wiki/) and an article [Contributing to Flutter:
+Wiki](https://github.com/flutter/flutter/wiki/), a video [How to contribute to Flutter](https://www.youtube.com/watch?v=4yBgOBAOx_A) on the [Flutter](https://www.youtube.com/c/flutterdev/) YouTube channel and an article [Contributing to Flutter:
```

Comment:
need a comma after "channel"

Reply:
should we just inline the video?

Reply:
Yeah, that sounds better.

Reply:
My next question would be where?

Reply:
GitHub Markdown does not support inlining the video ([link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#youtube-videos)). There is an alternative, which is not bad.